### PR TITLE
add arbitrum-nitro fees

### PR DESCRIPTION
**Pull request by dtmkeng**

### CODE REVIEW

**Comment by llamatester** [comment]
The arbitrum-nitro.ts adapter exports: 
        
 
 _______________________________________
    Fees for 6/11/2022
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 1/12/2019
    Timestamp: 1667779198
    Total fees: undefined
    Daily fees: 0
    Daily revenue: 0
    Total revenue: 0
    
    
    
    
    Done in 6.87s.

**Comment by llamatester** [comment]
The arbitrum-nitro.ts adapter exports: 
        
 
 _______________________________________
    Fees for 6/11/2022
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 1/12/2019
    Timestamp: 1667779198
    Total fees: undefined
    Daily fees: 0
    Daily revenue: 0
    Total revenue: 0
    
    
    
    
    Done in 9.31s.

**Comment by llamatester** [comment]
The arbitrum-nitro.ts adapter exports: 
        
 
 _______________________________________
    Fees for 6/11/2022
    _______________________________________
    
    ARBITRUM 👇
    Backfill start time: 29/5/2021
    Timestamp: 1667779198
    Total fees: undefined
    Daily fees: 0
    Daily revenue: 0
    Total revenue: 0
    
    
    
    
    Done in 7.65s.

File: fees/arbitrum-nitro.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            getWithdrawalTxs(CONGESTION_FEES, todaysBlock, yesterdaysBlock),
+          ]);
+          const withdrawalFee = sequesnerFee + infraFee + congestionFee;
+
+          const pricesObj = await getPrices(["coingecko:ethereum"], todaysTimestamp);
+          const dailyFees = (withdrawalFee) * pricesObj["coingecko:ethereum"].price
+
+          return {
+              timestamp,
+              totalFees: undefined,
```
Comment (0xngmi) [inline fees/arbitrum-nitro.ts]: better to delete this line rather than literally set it to undefined

**Comment by llamatester** [comment]
The arbitrum-nitro.ts adapter exports: 
        
 
 _______________________________________
    Fees for 6/11/2022
    _______________________________________
    
    ARBITRUM-NITRO 👇
    Timestamp attempted: 5/11/2022
    TypeError: data.reduce is not a function
        at getWithdrawalTxs (/home/runner/work/adapters/adapters/fees/arbitrum-nitro.ts:23:15)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async Promise.all (index 0)
        at async fetch (/home/runner/work/adapters/adapters/fees/arbitrum-nitro.ts:35:59)
        at async /home/runner/work/adapters/adapters/adapters/utils/runAdapter.ts:30:52
    
    
    
    
    Done in 7.97s.

File: fees/arbitrum-nitro.ts (hunk 1 above)
Comment (0xngmi) [reply fees/arbitrum-nitro.ts]: fixed

**Comment by 0xngmi** [comment]
my last update was wrong, sorry

i wouldn't list arbitrum nitro for now

**Comment by Rezacb0413** [comment]
>

**Review by Rezacb0413 — APPROVED** [review]

**Review by Rezacb0413 — APPROVED** [review]

**Comment by g1nt0ki** [comment]
@dtmkeng can you update the adapter code to use the latest helper methods and new nova contracts? https://docs.arbitrum.io/for-devs/dev-tools-and-resources/chain-info

if it takes too much time, we can close this PR for now

**Comment by g1nt0ki** [comment]
merged https://github.com/DefiLlama/dimension-adapters/pull/1976

**Pull request closed without merging**